src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,74 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import core.basesyntax.dao.StorageDao;
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+    private static User firstUser;
+    private static User secondUser;
+    private static User threeUser;
+    private static User fourthUser;
+
+    @BeforeAll
+    static void beforeAll() {
+        registrationService = new RegistrationServiceImpl();
+        firstUser = new User();
+        secondUser = new User();
+        threeUser = new User();
+        fourthUser = new User();
+    }
+
+    @Test
+    void registerUser_Ok() {
```

Comment:
the same for other test methods
![Screenshot 2022-07-06 at 10 37 02](https://user-images.githubusercontent.com/60258373/177499751-601d3a7e-8cde-4181-a772-00cfb20e7a9b.png)